### ISSUE TITLE
Rails/HttpPositionalArguments does not recognize include Rack::Test::Methods

### ISSUE THREAD

**Issue by jcraigk** [rubocop-hq/rubocop#7264]
#7095  Expected behavior

In RSpec tests that use `Rack::Test::Methods`, the `Rails/HttpPositionalArguments` cop should not warn about using keyword arguments, as that is not how the `Rack::Test::Methods` work.

## Actual behavior

In RSpec tests that include the line `include Rack::Test::Methods` and then use the `get`, `post`, etc methods supplied in that module, the `Rails/HttpPositionalArguments` mistakenly warns that keyword arguments should be used.  However, the `Rack::Test::Methods` do not support keyword arguments.

From the Rubocop docs:
```
# bad
get :new, { user_id: 1}

# good
get :new, params: { user_id: 1 }
```

## Steps to reproduce the problem

Run the `Rails/HttpPositionalArguments` on an RSpec file that includes the line `include Rack::Test::Methods` and then uses `get` with at least one parameter.

## RuboCop version

```
$ rubocop -V
0.74.0 (using Parser 2.6.3.0, running on ruby 2.6.3 x86_64-darwin17)
```

Tangentially related to https://github.com/rubocop-hq/rubocop/issues/3629
